src/__Private/LSPImpl/CodeActionCommand.hack

```diff
@@ -39,7 +39,7 @@ final class CodeActionCommand extends LSPLib\CodeActionCommand {
             return null;
           }
 
-          $linter = $e->getLinter();
+          $linter = $e->getLintRule();
           if (!$linter is AutoFixingLinter<_>) {
```

Comment:
`hh_client --lint`'s output does have fields for replacement, so could also be autofixing. I don't know if any linters currently do this, but it's definitely a future possibility, so `AutoFixingLinter` should probably also be split up to be a property of lint errors (or maybe rules) rather than the linters themselves

Definitely a separate PR for that though

Reply:
Will do! Thank you!